models/TradingAccount.py

```diff
@@ -103,11 +98,12 @@ def getOrders(self, market='', action='', status='all'):
                 if market == '':
                     return self.orders
                 else:
-                    return self.orders[self.orders['market'] == market]             
+                    return self.orders[self.orders['market'] == market]
         if self.app.getExchange() == 'coinbasepro':
             if self.mode == 'live':
                 # if config is provided and live connect to Coinbase Pro account portfolio
-                model = CBAuthAPI(self.app.getAPIKey(), self.app.getAPISecret(), self.app.getAPIPassphrase(), self.app.getAPIURL())
+                model = CBAuthAPI(self.app.getAPIKey(), self.app.getAPISecret(), self.app.getAPIPassphrase(),
```

Comment:
its usually bette to have either, everything in the same line, or one parameter per line.
I guess here we should have
```python
      CBAuthAPI(
        self.app.getAPIKey(),
        self.app.getAPISecret(),
        self.app.getAPIPassphrase(),
        self.app.getAPIURL()
      )
```